notebooks/data-sources/TestGrid/metrics/persistent_failures_analysis.ipynb

```diff
@@ -17,7 +17,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">&nbsp;perc_edges = (len(current_test["statuses"]) - 1) / n_cells
</pre><p>IIUC, this equation does not strictly capture the "percentage of pass to fail transition and vice versa" the issue is requesting. Since the statuses can take on any of the values defined in the class "TestStatus" this value would also include all of the occurrences of "fail to flake" transition as well as "pass to not run". I think you need to do a little more work here to get the desired metrics. Also, since there are other possible transitions, pass to fail and fail to pass are not inverse of each other and should be calculated as 2 different metrics.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Makes sense, updated!